framework/atoms/forms/text-inputs/text-inputs.html

```diff
@@ -8,14 +8,14 @@
   <div class="form-group focus">
     <label for="example-input-id-2">Text Input - active + focus</label>
     <p class="help-block">This is some placeholder help text.</p>
-    <input type="text" name="field-name-3" id="example-input-id-2" class="form-control is-focused"
+    <input type="text" name="field-name-2" id="example-input-id-2" class="form-control is-focused"
```

Comment:
As a small comment here, I would personally name these more meaningfully like example-input-id-focus, example-input-id-disabled, etc. Just for the sake of now wondering what these numbers are.
Not a stopper by any way.